content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -206,72 +147,129 @@ of when a :guilabel:`URL Regex` action is triggered (e.g., a page visit).
-shortcode for instant customer/supplier access to a live chat window.
+The :guilabel:`Widget` tab on the live chat channel form offers a website widget that can be added
+to third party platforms. Additionally, a short code is available, that can provide instant access
+to a live chat window.
 
 The live chat :guilabel:`Widget` can be applied to websites created through Odoo by navigating to
-the :menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Live
-Chat` section, and select the channel to add to the site. Click :guilabel:`Save` to apply.
+the :menuselection:`Website app --> Configuration --> Settings --> Email & Marketing`. Then scroll
+to the :guilabel:`Live Chat` section, and select the channel to add to the site. Click
+:guilabel:`Save` to apply.
 
-To add the widget to a website created on a third-party platform, click :guilabel:`COPY` and paste
+To add the widget to a website created on a third-party platform, click :guilabel:`Copy` and paste
 the code into the `<head>` tag on the site.
 
-Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`COPY`
+Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`Copy`
 button which contains a link to join directly.
 
 Create a support ticket from a live chat session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once live chat is enabled, operators will be able to communicate with site visitors in real time.
+Once live chat is enabled, operators are able to communicate with site visitors in real time.
 
 During the conversation, an operator can use the shortcut :doc:`command
-</applications/websites/livechat/responses>` `/helpdesk` to create a ticket without leaving
-the chat window. The transcript from the conversation will be added to the new ticket, under the
+</applications/websites/livechat/responses>` `/ticket` to create a ticket without leaving the chat
+window. The transcript from the conversation is added to the new ticket, under the
 :guilabel:`Description` tab.
 
+.. _receiving_tickets/website-form:
+
+Website Form
+------------
+
+Enabling the *Website Form* setting adds a new page to the website with a customizable form. A new
+ticket is created once the required form fields are filled out and submitted.
+
+To activate the website form, navigate to a team's settings page under :menuselection:`Configuration
+--> Helpdesk Teams`. Find the :guilabel:`Website Form` feature under the :guilabel:`Help Center`
+section, and check the box.
+
+After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
+:guilabel:`Teams` settings page to view and edit the new website form, which is created
+automatically by Odoo.
+
+.. note::
+   After enabling the website form, the *Teams* settings page may need to be refreshed before the
+   *Go to Website* smart button appears.
+
```

Comment:
How do I manually add a form and link it to the helpdesk?

Reply:
while this is *technically* possible (you can create a form and change the action on the form to "submit a ticket" instead of "send an email", it is actually a lot more work and leaves more room for error than just enabling it through the helpdesk.